frontend/src/components/navigation/Header.tsx

```diff
@@ -22,6 +22,9 @@ const Header = () => {
     errors.forEach(error => dispatch(clear(error.name)));
     setShow(false);
   };
+  const url = `${window.location.protocol}//${window.location.hostname}${
```

Comment:
Clicking the image should go to the home page

Reply:
this will force refresh the app, you likely want history.push here instead.